include/odp/api/spec/traffic_mngr.h

```diff
@@ -247,6 +247,75 @@ typedef struct odp_tm_queue_stats_capability_t {
 	};
 } odp_tm_queue_stats_capability_t;
 
+/** TM node specific statistics counters */
+typedef struct odp_tm_node_stats_t {
+	/** Packets dropped by this node post scheduling/shaping at this node */
+	uint64_t discards;
+
+	/** Octets dropped after scheduling/shaping at this node */
+	uint64_t discard_octets;
+
+	/** Green packets that are sent through this tm node */
```

Comment:
E.g. `Number of successfully processed green packets.` Same goes for all the colors.